File: app/src/main/java/com/victoria/bleled/util/architecture/AbsentLiveData.java
package com.victoria.bleled.util.architecture;

import androidx.lifecycle.LiveData;

public class AbsentLiveData<T> extends LiveData<T> {
    AbsentLiveData() {
        postValue(null);
    }

    public static AbsentLiveData create() {
        return new AbsentLiveData();
    }
}

File: libMBase/src/main/java/com/mario/lib/base/architecture/AbsentLiveData.kt
package com.mario.lib.base.architecture

import androidx.lifecycle.LiveData

class AbsentLiveData<T> internal constructor() : LiveData<T>() {
    init {
        postValue(null)
    }

    companion object {
        @JvmStatic
        fun <T> create(): AbsentLiveData<T> {
            return AbsentLiveData()
        }
    }
}